src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/EngineGeneralTest.java

```diff
@@ -1282,6 +1284,50 @@ public void minTest() throws DatabaseEngineException {
         assertEquals("result ok?", 0, (int) query.get(0).get("MIN").toInt());
     }
 
+    @Test
+    public void floorTest() throws DatabaseEngineException {
+        test5Columns();
+
+        EntityEntry.Builder entry = entry()
+                .set("COL1", 2)
+                .set("COL2", false)
+                .set("COL3", 2.5D)
+                .set("COL4", 3L)
+                .set("COL5", "ADEUS");
+
+        for (int i = 0; i < 10; i++) {
+            entry.set("COL1", i);
+            engine.persist("TEST", entry
+                    .build());
+        }
+
+        List<Map<String, ResultColumn>> query = engine.query(select(floor(column("COL3")).alias("FLOOR")).from(table("TEST")));
+
+        assertEquals("result ok?", 2.0, query.get(0).get("FLOOR").toDouble(), 0.1);
```

Comment:
`0.1` is a very high delta.

Ideally there should be a static variable to be used in all tests but please check the STDDEV tests that are using `0.0001D` as delta.

Reply:
For reference, we usually do this:

```
public static final Double DELTA = 1e-7;
```